doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -850,6 +856,67 @@ storages. You can change the size of the allocated memory using the
 error can also be resolved by increasing the size of the physical
 cluster disk.
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+CrashLoopBackOff status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pods do not start and have the status ``CrashLoopBackOff``
+
+.. code-block:: console
+
+   $ kubectl -n tarantool get pods
+   ---
+   NAME                                 READY   STATUS             RESTARTS   AGE
+   routers-0-0                           0/1     CrashLoopBackOff   6          8m4s
+   storages-0-0                          0/1     CrashLoopBackOff   6          8m4s
+   tarantool-operator-b54fcb6f9-2xzpn    1/1     Running            0          12m 
+
+Doing a kubectl describe pod will give us more information on that pod:
+
+.. code-block:: console
+
+   $ kubectl -n tarantool describe pod routers-0-0
+   ---
+   Events:
+     Type     Reason            Age                    From               Message
+     ----     ------            ----                   ----               -------
+     ...
+     Normal   Pulling           39m                    kubelet, minikube  Pulling image "vanyarock01/test-app:0.1.0-1-g4577716"
+     Normal   Pulled            39m                    kubelet, minikube  Successfully pulled image "vanyarock01/test-app:0.1.0-1-g4577716"
+     Normal   Created           37m (x5 over 39m)      kubelet, minikube  Created container pim-storage
+     Normal   Pulled            37m (x4 over 39m)      kubelet, minikube  Container image "vanyarock01/test-app:0.1.0-1-g4577716" already present on machine
+     Normal   Started           37m (x5 over 39m)      kubelet, minikube  Started container pim-storage
+     Warning  BackOff           4m25s (x157 over 38m)  kubelet, minikube  Back-off restarting failed container
+
+We see that the container cannot start. Rather, the container starts, but after starting it stops due to an internal error. To understand what is happening to him, let's see his logs:
```

Comment:
Let's see it's logs.

And it's better to keep lines short, like, 80-100 characters.